pyscript.core/src/stdlib.js

```diff
@@ -37,7 +37,7 @@ const python = [
     "_path = None",
 ];
 
-const ignore = new Ignore(python, "./pyweb");
+const ignore = new Ignore(python, "-");
```

Comment:
I would be happy to remove that `ignore` code / logic if we don't need to ignore anything anymore ... I mean, we can keep it around or commented but if we don't need it and we want to be sure everything we land in stdlib works in both Pyodide and MicroPython, maybe we can remove it? It doesn't have to happen in here, just wondering what you think about it, thanks.